src/main/java/jenkins/plugins/openstack/compute/JCloudsCloud.java

```diff
@@ -523,7 +523,7 @@ public FormValidation doTestConnection(
         @Restricted(DoNotUse.class)
         @RequirePOST
         public FormValidation doCheckEndPointUrl(@QueryParameter String value) {
-            Jenkins.getInstance().checkPermission(Jenkins.ADMINISTER);
+            Jenkins.getActiveInstance().checkPermission(Jenkins.ADMINISTER);
```

Comment:
Hmm... `getActiveInstance()` is deprecated in modern versions of Jenkins (as is `getInstance()`) - `Jenkins.get()` is favored these days (ever since Jenkins version 2.98).
Rather than update this code with an already-outdated call, it may be a better idea to bump the minimum version of Jenkins required to 2.98 (we already require Java 8, so we can't _actually_ use Jenkins 1.642.3 anymore) and then call `Jenkins.get()` everywhere instead.
@olivergondza WDYT?

Reply:
Yeah, or we can wait with this particular change until we require such version. Agreed it makes little sense to change it back and forth.

Reply:
Well, so bump Jenkins version in pom.xml to 2.98 is ok for all of you?

Reply:
Personally, I can't think of any reason _not_ to bump the minimum Jenkins version to 2.98 ... but I do not consider myself an expert on the pros & cons of making such changes (not knowing "why not" is not the same as knowing "why") ... and it's not my decision ;-)

What I would suggest, however, is that bumping the Jenkins version to 2.98 is not something that should be done as part of this PR - it would be better done as a separate PR, because it's a separate, significant, change in its own right.
If that separate PR gets merged then this PR can re-base and remove any changes mentioning `Jenkins.getActiveInstance()` as they'll all be `Jenkins.get()`.

Reply:
Agree, made PR#240